source/base/auto_derivative_function.cc

```diff
@@ -47,6 +47,19 @@ template <int dim>
 void
 AutoDerivativeFunction<dim>::set_formula (const DifferenceFormula form)
 {
+  // go through all known formulas, reject ones we don't know about
+  // and don't handle in the member functions of this class
+  switch (formula)
```

Comment:
oh dear, yes. I should learn to run the testsuite even for minor changes such as this. It's just soo painful to do the whole thing on my laptop...